shell/platform/common/text_editing_delta.h

```diff
@@ -0,0 +1,72 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_COMMON_TEXT_EDITING_DELTA_H_
+#define FLUTTER_SHELL_PLATFORM_COMMON_TEXT_EDITING_DELTA_H_
+#define _SILENCE_CXX17_CODECVT_HEADER_DEPRECATION_WARNING
```

Comment:
The use of `codecvt_utf8_utf16` caused a deprecation warning on Windows that wanted me to use `wstring`, but I've silenced it since this code needs to work on multiple platforms.

Reply:
Yep; we enable this in a few places. That said, instead of defining it here, please define this in shell/platform/common/BUILD.gn as we do [here](https://github.com/flutter/engine/blob/bba38c0b83122ec9b174271e74499d739416208b/shell/platform/common/BUILD.gn#L54-L57)

Reply:
Isn't that definition you linked exactly where I'd need to define this for this file?  text_editing_delta.h and cc are included in that block in this PR.  Or am I misunderstanding?  I still seemed to get a failure in the Windows tests caused by this.

Reply:
Ah yeah -- looks like that is indeed the exact target. Will try patching this in to see if I can repro.